rs/ethereum/cketh/minter/src/eth_rpc_client/mod.rs

```diff
@@ -22,6 +22,12 @@ pub mod responses;
 #[cfg(test)]
 mod tests;
 
+// This constant is our approximation of the expected header size.
+// The HTTP standard doesn't define any limit, and many implementations limit
+// the headers size to 8 KiB. We chose a lower limit because headers observed on most providers
+// fit in the constant defined below, and if there is spike, then the payload size adjustment
+// should take care of that.
+pub const HEADER_SIZE_LIMIT: u64 = 2 * 1024;
```

Comment:
nit: I have the impression it could be private
```suggestion
const HEADER_SIZE_LIMIT: u64 = 2 * 1024;
```